src/components/Controls.vue

```diff
@@ -119,6 +129,17 @@ export default Vue.extend({
         store.commit.setDirectionalEdges(value);
       },
     },
+
+    network(): Network | null {
+      return store.getters.network;
+    },
+
+    searchItems(): string[] {
```

Comment:
This function has an odd name. It's not searching the items, it's returning a candidate pool--is that right?

I'd suggest a better name for this function to avoid confusion for future devs.

Reply:
Fixed in 102c6d8